examples/file-share-svelte/README.md

```diff
@@ -0,0 +1,54 @@
+# File Share (Svelte)
```

Comment:
For consistency's sake, can you please change to:

File share example with Jazz and Svelte 

I'm also happy to change this. I just want to be consistent.